src/ui/splat-list.ts

```diff
@@ -0,0 +1,129 @@
+import { Container, Label, Element as PcuiElement } from 'pcui';
```

Comment:
Why is `Element as PcuiElement` even necessary?

Reply:
It was necessary when SplatList was defined in control-panel.ts, no more.

Updated - thanks!